NOTICE

```diff
@@ -0,0 +1,4 @@
+Voice Metadata Standard - FINOS
+Copyright 2018 Fintech Open Source Foundation info@finos.org
```

Comment:
@finos-admin FINOS owns the copyright here right?

Reply:
@vitorespindola - The NOTICE file must contain the copyright notices of each copyright holder who has contributed to the project - you can read more on https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530375/Contribution+Compliance+Requirements#ContributionComplianceRequirements-TheNOTICEfile